### ISSUE TITLE
Remove 'unsafe-eval' from CSP

### ISSUE THREAD

**Issue by fregante** [pixiebrix/pixiebrix-extension#4387]
'unsafe-eval' is not allowed in MV3’s manifest:

<img width="254" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/193245201-1c703301-bcfa-437c-85e3-0985f20f4d21.png">

Dropping it resurfaces:

- https://github.com/pixiebrix/pixiebrix-extension/issues/2946

<img width="402" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/193244833-f6d10ba6-a509-4d06-b72c-ae4a5f9ce511.png">

## Tasks

- [ ] https://github.com/pixiebrix/pixiebrix-extension/issues/3059
- [x] https://github.com/pixiebrix/pixiebrix-extension/issues/4730
- [x] https://github.com/pixiebrix/pixiebrix-extension/issues/105

### Our options

 
> 1. Get RJSF to add support for an eval-less validation library (either by asking nicely or by submitting as PR). We use:  [npmjs.com/package/@cfworker/json-schema](https://www.npmjs.com/package/@cfworker/json-schema). (I had actually looked at AJV previously but didn't use it because of eval)
> 2. Move schema validation to the Chrome sandbox: [Move template engine and brick evaluation to Chrome sandbox #105](https://github.com/pixiebrix/pixiebrix-extension/issues/105). This also would probably require modifying RJSF to support a custom validation function. This approach is nice because it also makes sense to move Nunjucks and JQ over there
> 3. Modify AJV to not require Function

**Comment by fregante** [comment]
I think this is already covered by our webpack MV3 upgrade script:

https://github.com/pixiebrix/pixiebrix-extension/blob/cc77c2126d9a036b6fa6b4e7ed6d4ad6319fc0b1/webpack.config.js#L171

Once the RJSF fix is done, this won't require further changes.
